processor/src/test/kotlin/com/tobrun/datacompat/DataCompatProcessorTest.kt

```diff
@@ -27,8 +28,11 @@ interface EmptyInterface2
 @Deprecated
 @DataCompat
 private data class PersonData(
+    @Default("\"John\"")
     val name: String,
-    val nickname: String? = null,
+    @Default("null")
+    val nickname: String?,
+    @Default("23")
```

Comment:
nit:
maybe add `DefaultString`, `DefaultInt`, ... ?

Reply:
how will you handle e.g. `CustomClass` type?

Reply:
we will not cover all the strong types but string representation allows that (and does not allow any strict validations but everything has its price)